app/helpers/admin_helper.rb

```diff
@@ -64,6 +64,16 @@ def admin_can_update_user_roles?
     policy(User).permitted_attributes.include?(roles: [])
   end
 
+  def admin_can_edit_user_role(role)
```

Comment:
Instead of a separate helper, let's keep this logic in the UserPolicy class. You can define a method `can_edit_user_role?(role)` there. Then in the view, you can do:
```rb
policy(User).can_edit_user_role?(role)
```

Reply:
done